src/pages/guides/[...slug].astro

```diff
@@ -0,0 +1,23 @@
+---
+import { getCollection } from 'astro:content';
+import Layout from '../../components/Layout.vue';
+import BaseLayout from '../../layouts/BaseLayout.astro';
+import customComponents from '../../utils/customComponents';
+
+export async function getStaticPaths() {
```

Comment:
To [generate dynamic routes during build time](https://docs.astro.build/en/core-concepts/routing/#static-ssg-mode), we need to explicitly declare them